db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Genre.create(name: "Arts")
```

Comment:
We want our seed script to be _noisy_ so I would favor the use of `create!` instead of `create` so we know when seeding fails.